src/rrt/Tree.hpp

```diff
@@ -309,7 +350,21 @@ class Tree {
         }
 
         // Add a node to the tree for this state
+        int lengthT = sizeof(intermediateState) / sizeof(0.0);
```

Comment:
Assumes state is a double. if you want to make this more general, `sizeof(intermediateState) / sizeof(intermediateState[0])` works, but that assumes intermediateState has at least length 1. This is an old C way of doing things though.

For C++11, you can do [std::extent](http://en.cppreference.com/w/cpp/types/extent)